README.md

```diff
@@ -267,7 +267,6 @@ cmake flags:
 -DTRITON_COMMON_REPO_TAG=r21.10
 -DTRITON_THIRD_PARTY_REPO_TAG=r21.10
 -DTRITON_CORE_REPO_TAG=r21.10
--DTRITON_BACKEND_REPO_TAG=r21.10
```

Comment:
is this intentional?

Reply:
yep this line is not needed in the build